packages/xo-web/src/common/intl/messages.js

```diff
@@ -148,7 +148,7 @@ const messages = {
   xoa: 'XOA',
 
   // ----- Support -----
-  noSupport: 'No support',
+  noSupport: 'No Support',
```

Comment:
Why adding uppercase for support here?

Reply:
Because it lines up with everything else that is two words.

Reply:
It seems the majority of translations I can see in this file with 2 words have only 1 caps on the first word, except for acronyms, like DR for Disaster Recovery.

Reply:
So this means for some lines left with 2 caps are the one we should replace (we could count but I have the feeling we got far more 1 caps in 2 words than the opposite)